README 2.md

```diff
@@ -0,0 +1,119 @@
+# Codacy ESLint
```

Comment:
Just a minor thing, could we merge the README2 with README instead of having both files?

Reply:
I've merged into one single file, please check if it sounds good to you. I removed things related to ESlint that are not required for this plugin I think.